nf_core/modules/update.py

```diff
@@ -453,13 +448,13 @@ def get_all_modules_info(self, branch=None):
         # don't try to update those that don't
         i = 0
         while i < len(modules_info):
-            repo, module, _ = modules_info[i]
+            repo, module, sha = modules_info[i]
             if not repo.module_exists(module):
                 log.warning(f"Module '{module}' does not exist in '{repo.fullname}'. Skipping...")
                 modules_info.pop(i)
             elif sha is not None and not repo.sha_exists_on_branch(sha):
                 log.warning(
-                    f"Git sha '{sha}' does not exists on the '{branch}' of '{repo.fullname}'. Skipping module '{mod}'"
+                    f"Git sha '{sha}' does not exists on the '{repo.branch}' of '{repo.fullname}'. Skipping module '{mod}'"
```

Comment:
If I am not mistaken, `mod` should be `module`

```suggestion
                    f"Git sha '{sha}' does not exists on the '{repo.branch}' of '{repo.fullname}'. Skipping module '{module}'"
```